pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = --cov ./deepparse --cov-report html --cov-report xml --cov-config=.coveragerc
+#addopts = --cov ./deepparse --cov-report html --cov-report xml --cov-config=.coveragerc
```

Comment:
Reason for this?

Reply:
Forgot to remove, you cannot debug tests without this line commented